src/main/java/com/prgrms/needit/domain/contract/controller/ContractController.java

```diff
@@ -0,0 +1,91 @@
+package com.prgrms.needit.domain.contract.controller;
+
+import com.prgrms.needit.common.response.ApiResponse;
+import com.prgrms.needit.domain.contract.controller.bind.ContractRequest;
+import com.prgrms.needit.domain.contract.entity.response.ContractResponse;
+import com.prgrms.needit.domain.contract.entity.response.DonationContractResponse;
+import com.prgrms.needit.domain.contract.entity.response.WishContractResponse;
+import com.prgrms.needit.domain.contract.service.ContractService;
+import java.net.URI;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/contract")
+@RequiredArgsConstructor
+public class ContractController {
+
+	private final ContractService contractService;
+
+	@GetMapping("/donation/{donationId}/{contractId}")
```

Comment:
```suggestion
	@GetMapping("/{contractId}/donation/{donationId}")
```
관련 리소스 뒤에 해당 아이디를 붙이는게 RESTful 한 것 같아요!

Reply:
뭔가 contract에 속한 donation 을 다루는 것처럼 느껴지진 않을까요?

Reply:
그럼 API 설계에 대해서 다시 고민해봐야 할 것 같아영 위 API처럼 작성하면 restful 하지 않다는 것으로 알고있거든용 [참고](https://eblo.tistory.com/44)